test/unit/models/auctions.test.js

```diff
@@ -0,0 +1,113 @@
+const chai = require("chai");
+const sinon = require("sinon");
+const { expect } = chai;
+
+const cleanDb = require("../../utils/cleanDb");
+const firestore = require("../../../utils/firestore");
+const addUser = require("../../utils/addUser");
+
+// Import models
+const auctions = require("../../../models/auctions");
+const walletsQuery = require("../../../models/wallets");
+const auctionModel = firestore.collection("auctions");
+const bidModel = firestore.collection("bids");
+
+// Import fixtures
+const { auctionData } = require("../../fixtures/auctions/auctions");
+const user = require("../../fixtures/user/user")();
+const currencies = require("../../fixtures/currencies/currencies");
+
+const { initial_price: initialPrice, item_type: itemType, end_time: endTime, quantity } = auctionData;
+
+describe("auctions", function () {
+  let seller, auctionId, bidder;
+
+  beforeEach(async function () {
+    seller = await addUser();
+    bidder = await addUser(user[1]);
+    auctionId = await auctions.createNewAuction({ seller, initialPrice, itemType, endTime, quantity });
+  });
+  afterEach(async function () {
+    await cleanDb();
+    sinon.restore();
+  });
+
+  describe("createNewAuction", function () {
```

Comment:
Can we go ahead with simple English words with spaces here? Rather than camelCase

Reply:
Shouldn't it be the function name as done in other unit tests?